avsomr/avsomrnetrecognizer.h

```diff
@@ -31,6 +33,6 @@ class AvsOmrNetRecognizer
     QNetworkAccessManager* _net{nullptr};
 };
 
-}
+}}
```

Comment:
??

Reply:
this is the closure of two namespaces

Reply:
which start on different lines, so should end on different lines too, possible with a comment?

Reply:
placed on different lines and added comments